tests/setup-jest.js

```diff
@@ -0,0 +1,37 @@
+import $ from 'jquery'
+global.$ = $
+global.jQuery = $
+
+const langFiles = require('formbuilder-languages')
+global.FB_EN_US = langFiles['en-US']
+
+const xhrMockClass = () => ({
+  open            : jest.fn(), //(method, url) => {console.log(method + ':' + url)},
+  send            : jest.fn().mockImplementation(function(){ this.onload() }),
+  setRequestHeader: jest.fn(),
+  abort: jest.fn(),
+  addEventListener: jest.fn(),
+  dispatchEvent: jest.fn(),
+  error: jest.fn(),
+  getAllResponseHeaders: jest.fn(),
+  getResponseHeader: jest.fn(),
+  //load: jest.fn(),
```

Comment:
do we need this?

Reply:
So far yes.

The jQuery part sets up jQuery into the global space

Since I am testing the raw .js files rather than the built files I need to configure FB_EN_US to set the language (what webpack normally does)

I needed the xhrMockClass when testing loading of script etc. We can likely adjust in the future but mocking these helps